cfde_submit/main.py

```diff
@@ -223,4 +217,40 @@ def logout():
 
 @cli.command(name='version')
 def version_cmd():
+    """ Output version information and exit """
     click.secho(version.__version__)
+
+
+@cli.command()
+def reset():
+    """ Reset cfde-submit configuration """
+    remove = yes_or_no("Would you like to reset your cfde-submit settings and submit history?")
+    if remove:
+        if os.path.exists(DEFAULT_STATE_FILE):
+            os.remove(DEFAULT_STATE_FILE)
+        else:
+            sys.exit("No cfde-submit settings exist, skipping")
```

Comment:
That's a nice simple solution for clearing config options. This also makes me notice there are two config files, one for setting and another in the client here for tokens: https://github.com/nih-cfde/cfde-submit/blob/main/cfde_submit/client.py#L20. It may be good to merge those at some point.